### DISCUSSION TITLE
Maga trump on sol

### DISCUSSION THREAD

**Discussion by MagaMemeCoinsolana** [phantom/docs#169] [General]
Contract address: AuxATgVcR8MwNRcGQVkitqzmc7932BJXEoKNtno1BJLN

Ticker: MAGA
Name: MAGA TRUMP ON SOL

![9E3A2920-9D4E-4B3F-9518-5F7A5EACDA76](https://github.com/phantom/docs/assets/158356922/331b5e3e-e079-4a2e-8ded-756846106ac2)

**Comment by adamdelphantom** [comment]
Hello @MagaMemeCoinsolana,

It appears the uri to the token image points to a broken link: 
https://pbs.twimg.com/profile_images/1728804657047408640/awhPAnK3_400x400.jpg

**Comment by MagaMemeCoinsolana** [comment]
Hello sorry for late response.

Here is the logo and updated token name

Name: MAGA Meme Coin
Ticker: $MAGA

**Comment by MagaMemeCoinsolana** [comment]
Here is link to logo

https://images.squarespace-cdn.com/content/v1/657db384f1128d514a0862e3/4f00e0db-89fc-4f16-8682-3507c7c000d3/photo_2024-02-19_13-10-32.jpg?format=500w

**Reply by adamdelphantom** [reply]
@MagaMemeCoinsolana This appears to be rendering as expected in Phantom as expected, now.
